gef-ui/server/controllers/amendments/helpers/navigation.helper.ts

```diff
@@ -61,7 +65,9 @@ export const getPreviousPage = (currentPage: PortalAmendmentPage, amendment: Por
     throw new Error(`Cannot get previous page for ${currentPage}`);
   }
 
-  return `/gef/application-details/${amendment.dealId}/facilities/${amendment.facilityId}/amendments/${amendment.amendmentId}/${journey[currentPageIndex - 1]}`;
+  const { dealId, facilityId, amendmentId } = amendment;
+
+  return getAmendmentsUrl({ dealId, facilityId, amendmentId, page: journey[currentPageIndex - 1] });
```

Comment:
Can `page` be declared before this call and pass `page` just as an argument please.

Reply:
Yep have changed